src/libraries/System.Runtime.Extensions/tests/System/Random.cs

```diff
@@ -178,6 +202,30 @@ public void Next_LongLong_AllValuesWithinSmallRangeHit(bool derived, bool seeded
             Assert.DoesNotContain(44L, hs);
         }
 
+        public static IEnumerable<object[]> Next_LongLong_Next_IntInt_AllValuesAreWithinRange_MemberData() =>
+            from derived in new[] { false, true }
+            from seeded in new[] { false, true }
+            from (long min, long max) pair in new[]
+            {
+                (1L, 2L),
+                (0L, long.MaxValue),
+                (2147483648, 2147483658),
+                (-1L, long.MaxValue),
+                (long.MinValue, 0L),
+            }
+            select new object[] { derived, seeded, pair.min, pair.max };
+
+        [Theory]
+        [MemberData(nameof(Next_LongLong_Next_IntInt_AllValuesAreWithinRange_MemberData))]
+        public void Next_LongLong_Next_IntInt_AllValuesAreWithinRange(bool derived, bool seeded, long min, long max)
+        {
+            Random r = Create(derived, seeded);
+            for (int i = 0; i < 100; i++)
```

Comment:
If we had a bug, would this test realistically discover it? 
I assume that 100 could be quite a bit larger without significant time cost.

Reply:
We did have a bug,  and it typically only took two iterations to flag it.  I could make it larger if you like,  though.

Reply:
I don't feel strongly